### DISCUSSION TITLE
Spanish translation

### DISCUSSION THREAD

**Discussion by k3ssdev** [bcholmes/StarTrek2d20#215] [General]
Hi there,

I like this app very much, and I'd like to collaborate with the Spanish translation. As I've seen in the translations folder, there is a CSV file, so it is as simple as filling up the table in the proper column, right?

I want to take this opportunity to thank you for the work you've done. Your application inspired me for my final programming course project, in which I made an Android app for Star Trek character sheets and dice rolling, although my app is much simpler.

Best,
Alberto

**Comment by mprangenberg** [comment]
You can help with the translation through a public google docs spreadsheet. See here for more information: https://github.com/bcholmes/StarTrek2d20/blob/sta-complete/translation/README.md

**Reply by k3ssdev** [reply]
Thanks! I'll help with that, I hope to contribute little by little.

**Reply by bcholmes** [reply]
I'd love it if someone fairly fluent with Spanish could help out with the Spanish translation. What mprangenberg says is completely true: add translations to the Google spreadsheet, and they'll be picked up when I do a next release.

Thanks for helping out!

**Reply by k3ssdev** [reply]
I'm currently working on it. The list of words and phrases is extensive...

**Reply by bcholmes** [reply]
Cool. I just uploaded a new version of the app which includes the translations that you've made to date.

**Reply by k3ssdev** [reply]
I've been making some changes (fixed some mistakes) and adding more lines. I think it could take 2-3 weeks to translate all the rows. 

Larga y próspera vida 🖖

**Reply by bcholmes** [reply]
Can I credit you on the credits page? As k3ssdev, or something else?

**Reply by k3ssdev** [reply]
Sure, you can mention me on credits page. You can link to my github page https://github.com/k3ssdev 

By the way, I'm working on some word and expression corrections in the translated items. For example Spaceframe is adapted as "Plantilla" for the ships and bases (it means like template). And Focus is translated as "Especialidad", as it is translated in french. These translations make more sense in spanish. And the last one, Momentum, it's translated as Inercia as the spanish edition was translated like that so the players can recognize the concept.

**Comment by k3ssdev** [comment]
It is about 60% completed  🖖
